src/components/HeaderMainPage.tsx

```diff
@@ -0,0 +1,24 @@
+import shoppingCart from '../images/shopping_cart_black_24dp 1.png';
+import React from 'react';
+import Logo from '../images/Logo-header.svg';
+import { useNavigate } from 'react-router-dom';
+import { Button } from '@mui/material';
+
+export const Header = () => {
```

Comment:
rename const to the name of the file -> HeaderMainPage and don't forget to rename in place where you use it